server/api/api/endpoints/nuclio.py

```diff
@@ -0,0 +1,22 @@
+import mlrun.common.schemas
+from mlrun.utils import logger
+import server.api.utils.clients.async_nuclio
+from server.api.api import deps
+from fastapi import (
+    APIRouter,
+    Depends,
+    Query,
+)
+
+router = APIRouter()
+
+
+@router.get("/nuclio/api-gateways")
+async def list_api_gateways(
+    auth_info: mlrun.common.schemas.AuthInfo = Depends(deps.authenticate_request),
+    namespace=Query(default=None, alias="namespace"),
+    project_name=Query(default=None, alias="project-name"),
```

Comment:
get it from path, like other endpoint do. this is how we reflect relationship between project and its resources.